near/omni-types/src/lib.rs

```diff
@@ -323,8 +323,16 @@ impl TransferMessage {
     }
 }
 
+#[derive(BorshDeserialize, BorshSerialize, Serialize, Deserialize, Debug, Clone)]
+pub enum PayloadPrefix {
```

Comment:
I'd called it PayloadType perhaps but up to you

Reply:
6d94fd3f342218bab869f1623a8edc5ba03506dc